app/models/measurement.rb

```diff
@@ -52,4 +52,64 @@ def self.find_by_perm_link(perm_link)
     perm_link = perm_link.sub('lmi_total_custom_', '').sub('lmi_total_', '')
     find_by(perm_link: ["lmi_total_#{perm_link}", "lmi_total_custom_#{perm_link}"])
   end
+
+  def load_gr_value(params)
```

Comment:
One trick you can that may make it slightly easier is to use `delegate` to delegate methods to the instance you're working with, e.g. see: https://github.com/CruGlobal/measurements-api/blob/staging/app/models/gr_sync/assignment_push.rb#L20